dobject.py

```diff
@@ -315,53 +317,53 @@ class HighScore:
     of a value and a score.  The user may suggest a new value and score.  If the new
     score is higher than the current score, then the value and score are updated.
     Otherwise, they are not.
-    
+
     The value can be any object, and the score can be any comparable object.
-    
+
     To ensure that serialization works correctly, the user may specify a
     translator function that converts values or scores to and from a format that
     can be serialized reliably by dbus-python.
-    
+
     In the event of a tie, coherence cannot be guaranteed.  If ties are likely
     with the score of choice, the user may set break_ties=True, which appends a
     random number to each message, and thereby reduces the probability of a tie
     by a factor of 2**52.
     """
+
     def __init__(self, handler, initval, initscore, value_translator=empty_translator, score_translator=empty_translator, break_ties=False):
         self._logger = logging.getLogger('stopwatch.HighScore')
         self._lock = threading.Lock()
         self._value = initval
         self._score = initscore
-        
+
         self._break_ties = break_ties
         if self._break_ties:
             self._tiebreaker = random.random()
         else:
             self._tiebreaker = None
-        
+
         self._val_trans = value_translator
         self._score_trans = score_translator
-        
+
         self._handler = handler
         self._handler.register(self)
-        
+
         self._listeners = []
-    
+
     def _set_value_from_net(self, val, score, tiebreaker):
         self._logger.debug("set_value_from_net " + str(val) + " " + str(score))
         if self._actually_set_value(val, score, tiebreaker):
             self._trigger()
-    
+
     def receive_message(self, message):
         self._logger.debug("receive_message " + str(message))
         if len(message) == 2: #Remote has break_ties=False
-            self._set_value_from_net(self._val_trans(message[0], False), self._score_trans(message[1], False), None)
+            self._list_value_from_net(self._val_trans(message[0], False), self._score_trans(message[1], False), None)
```

Comment:
You change _set_value_from_net to _list_value_from_net here and a few lines later, but the function referenced is not changed.

Reply:
Fixed on https://github.com/sugarlabs/stopwatch/pull/7/commits/0b3a93d263c37e29be58b9390d63a0c924c4c840 ,